sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -94,11 +94,36 @@
      * @param keyType The type of key to create. For valid values, see {@link KeyType KeyType}.
      * @throws ResourceModifiedException if {@code name} or {@code keyType} is null.
      * @throws HttpRequestException if {@code name} is empty string.
+     * @return A {@link Mono} containing the {@link Key created key}.
```

Comment:
I see an old code sample above. I assume you'll update these so I won't point them out any longer in this PR review.